src/snapshots/repl_basic_example.md

```diff
@@ -15,7 +15,7 @@ type=repl
 ---
 Evaluation error: error.LayoutError
 ---
-Evaluation error: error.LayoutError
+<scalar>
```

Comment:
Not a blocker, but I'd certainly like for this to print out the actual string literal instead of just `<scalar>`! 😄

Reply:
Added in commit 08e2f63332adc45e07f8ebdbc222c51a47b836bf